common/source/docs/common-blheli32-passthru.rst

```diff
@@ -1,62 +1,149 @@
+  .. image:: ../../../images/blhelisuite32.jpg
+    :target: ../_images/blhelisuite32.jpg
 
-- Use :ref:`SERVO_BLH_MASK <SERVO_BLH_MASK>` to enable BLHeli pass-through and telemetry support on non-multirotor motors and / or exactly specify which servo outputs you want to enable BLHeli pass-through and telemetry on.
+.. note::
+   ArduPilot firmware supports the pass-through protocol with up-to-date BLHeli32 firmware and BLHeliSuite32, or BLHeli_S firmware and BLHeliSuite only.
 
-- Set :ref:`SERVO_BLH_PORT <SERVO_BLH_PORT>` to specify the autopilot's port used to connect to your PC running BLHeliSuite32 (or BLHeliSuite for BLHeli_S) for ESC configuration. It defaults to USB and likely does not need to be altered. Beware that this does NOT specify the serial port used for the ESC's telemetry feedback to your autopilot!
+.. warning::
+   For pass-through to function, the autopilot must be configured to use one of the DShot protocols.  If you wish to eventually use one of the other protocols (e.g. PWM, OneShot125) that the ESC supports, you may still configure the ESCs using Pass-Through (e.g. change motor directions, set min/max values, etc) but then finally re-configure the autopilot to *not* use DShot.  Once the autopilot and ESCs are rebooted the ESC should auto-detect that the ESCs are no longer using DShot.
 
-Now connect a USB cable to your autopilot and use BLHeliSuite32 on Windows to connect. Select "BLHeli32 Bootloader (Betaflight/Cleanflight)" from the interfaces menu.
+..  youtube:: np7xXY_e5sA
+    :width: 100%
 
-.. image:: ../../../images/blhelisuite32.jpg
-    :target: ../_images/blhelisuite32.jpg
+.. _blheli32-reversible-dshot:
 
 Reversible DShot ESCs
 ---------------------
 
-Currently, only BLHeli32 and BLHeli-S capable reversible DShot ESCs are supported. In order to use one, the output which drives it must be designated with the appropriate bit in the :ref:`SERVO_BLH_3DMASK<SERVO_BLH_3DMASK>` bitmask parameter. This will map the outputs 1000-1500-2000 values to the correct digital values for the ESC to provide FullReverse-Idle-FullForward range operation, respectively.
+Currently, only BLHeli32 and BLHeli_S capable reversible DShot ESCs are supported. In order to use one, the output which drives it must be designated with the appropriate bit in the :ref:`SERVO_BLH_3DMASK<SERVO_BLH_3DMASK>` bitmask parameter. This will map the outputs 1000-1500-2000 values to the correct digital values for the ESC to provide FullReverse-Idle-FullForward range operation, respectively.
 
-If the craft has been setup for DShot commands then ArduPilot will supply the correct command at startup in order to set the ESCs in reversible mode.
+If :ref:`DShot commands <dshot-commands>` have been enabled then ArduPilot will automatically configure the ESCs to reversible mode at startup.  If DShot commands have not been configured you must manually configure the ESCs' "Motor Direction" to "Bidirectional 3D" as shown below.
```

Comment:
```suggestion
If :ref:`DShot commands <dshot-commands>` have been enabled then ArduPilot will automatically configure the ESCs to reversible mode (3D mode) at startup, according to the :ref:`SERVO_BLH_3DMASK<SERVO_BLH_3DMASK>`. Enabling :ref:`DShot commands <dshot-commands>`  will allow the other DShot commands to be sent to any other ESC configured as DShot by the DShot mask parameters discussed in :ref:`DShot setup instructions <common-dshot-escs>` . 

 Otherwise, you must manually configure the ESCs' "Motor Direction" to "Bidirectional 3D" as shown below.
```

Reply:
txs, updated the wording as suggested.